### ISSUE TITLE
Add event support to White

### ISSUE THREAD

**Issue by JakeGinnivan** [TestStack/White#140]
White supports automation properties, it might be good to support automation events

http://msdn.microsoft.com/en-us/library/ms752286.aspx
http://msdn.microsoft.com/en-us/library/ms748252.aspx

Would this be useful to anyone?

**Comment by ashishshuklabs** [comment]
Definitely. One of the many reasons could be improved logging. I'm currently using White interceptor to do this. However, events would greatly help in improving the code I've written up.
